babel.config.js

```diff
@@ -1,5 +1,13 @@
 module.exports = {
   presets: ['@babel/preset-env', '@babel/preset-react'],
-  plugins: ['preval'],
+  plugins: [
+    [
+      '@babel/transform-runtime',
```

Comment:
why this change?

Reply:
That was needed for the script to migrate to run. But i have seen many times an error in the console for the missing regenerator-runtime in certain situations, so i thought that this would also be the way to fix that. Unfortunately i don't have "proof" about this, because i cannot reproduce the issue i described.
So if you think there are downsides of using this plugin we can remove it for now and reintegrate it only if we experience again the issue i mentioned.

Reply:
For the script to run, ok. If it's not required after the work of the script and you're not sure about the benefits, I'd remove it, but it's up to you

Reply:
removed it @kalinchernev ;)